app/serializers/api/admin/reports/order_serializer.rb

```diff
@@ -0,0 +1,63 @@
+class Api::Admin::Reports::OrderSerializer < ActiveModel::Serializer
```

Comment:
Sorry, maybe I wasn't clear, I don't mean that ui-grids needs to receive JSON, I just mean that it takes data and renders it in the DOM